src/main/io/statusindicator.c

```diff
@@ -32,7 +32,9 @@ static uint32_t warningLedTimer = 0;
 typedef enum {
     WARNING_LED_OFF = 0,
     WARNING_LED_ON,
-    WARNING_LED_FLASH
```

Comment:
I rebased this PR after merging #2201.

I also fixed the dependency issues and code seperation issues in #2201 which I noted in the comments of #2201.

done.